test/parallel/test-http2-invalidheaderfield.js

```diff
@@ -0,0 +1,59 @@
+// spaced headers
```

Comment:
Is this a test description?
Then could you please move and clarify it a little (see lines 5-6 here https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#test-structure).

Reply:
Done.